README.rst

```diff
@@ -48,17 +48,26 @@ Adjust according to your operating system or personal preferences.
 #. Install a virtual environment for development (Python 2 or Python 3, you choose!)::
 
     $ sudo pip install virtualenvwrapper
+
+#. Follow the instructions from http://virtualenvwrapper.readthedocs.org/en/latest/install.html#basic-installation
+#. You will need to source the virtualenvwrapper.sh file for the following commands to work
+
     $ mkvirtualenv --python=python3 kolibri
     $ workon kolibri
 
-#. Install kolibri as an editable, meaning your installation will point to your git checkout::
-
+    #. Install kolibri as an editable, meaning your installation will point to your git checkout::
     $ pip install -e .
 
+#. Run the migration script to create and initialize the database
+   $ kolibri manage migrate
+
 #. Run the development server::
 
     $ kolibri manage runserver
 
+#. Install the pre-commit package: http://pre-commit.com/
```

Comment:
This is the wrong fix - what should be done for dev is to run:
```pip install -r requirements/dev.txt```
The pre-commit package requirement is included in there.